tools/build_protobuf

```diff
@@ -114,6 +114,6 @@ else
 fi
 
 $func core/src/trezor/messages "$CORE_PROTOBUF_SOURCES" "$CORE_MESSAGES_IGNORE" TRUE --no-init-py
-$func python/src/trezorlib/messages "$PYTHON_PROTOBUF_SOURCES" "$PYTHON_MESSAGES_IGNORE" FALSE -P ..protobuf
+$func python/src/trezorlib/messages "$PYTHON_PROTOBUF_SOURCES" "$PYTHON_MESSAGES_IGNORE" FALSE --include-deprecated -P ..protobuf
```

Comment:
Why do we want to keep deprecated fields in python-trezor?

Reply:
Because we are [using](https://github.com/trezor/trezor-firmware/blob/0600d87c8c2e910919c8093481901412fe2163f9/python/src/trezorlib/client.py#L180) it for some backwards compatibility (old passphrase design).

Reply:
OK